dataphos-persistor/templates/resubmitter.yaml

```diff
@@ -0,0 +1,95 @@
+{{- range $rsb, $target := .Values.resubmitter }}
+{{- $broker := get $.Values.brokers $target.broker -}}
+{{- $storage := get $.Values.storage $target.storage -}}
+{{- $indexer := (get ($.Values.indexer | default dict) $target.indexer | default dict) -}}
+{{- $isGcp := or (eq $broker.type "pubsub") (eq $storage.type "gcs") -}}
+{{- $isAzure := or (eq $broker.type "servicebus") (eq $storage.type "abs") -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: per-rsb-{{ $rsb }}-config
+  namespace: {{ $.Values.namespace }}
+data:
+  INDEXER_URL: "http://persistor-idx-api-svc-{{ $target.indexer }}.{{ $.Values.namespace }}.svc.cluster.local:8080"
```

Comment:
this should be **per**-idx-...

Reply:
Scratch that, it should be:

INDEXER_URL: "http://per-idx-api-/{{ $target.indexer }}-svc.{{ $.Values.namespace }}.svc.cluster.local:8080"